cloudant-spark-sql/src/main/scala/com/cloudant/spark/common/JsonStoreConfigManager.scala

```diff
@@ -119,7 +126,12 @@ import com.cloudant.spark.CloudantConfig
       
       val intSchemaSampleSize = calculateSchemaSampleSize(schemaSampleSize);
       
-      println(s"Use dbName=$dbName, indexName=$indexName, viewName=$viewName, $PARTITION_CONFIG=$total, $MAX_IN_PARTITION_CONFIG=$max, $MIN_IN_PARTITION_CONFIG=$min, $REQUEST_TIMEOUT_CONFIG=$requestTimeout,$CONCURRENT_SAVE_CONFIG=$concurrentSave,$BULK_SIZE_CONFIG=$bulkSize,$SCHEMA_SAMPLE_SIZE_CONFIG=$intSchemaSampleSize")
+      println(s"Use connectorVersion=$CLOUDANT_CONNECTOR_VERSION, dbName=$dbName, " +
```

Comment:
@ykye Kang Ye, Oh I understand what you were thinking. But here dbName and indexName are just variable names, they are not options provided to users, so that we have to keep them consistent.  That is why, if you don't mind, I will leave  'dbName' and 'indexName' as they are now.